openpype/hosts/max/plugins/create/create_redshift_proxy.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""Creator plugin for creating camera."""
+from openpype.hosts.max.api import plugin
+from openpype.pipeline import CreatedInstance
+
+
+class CreateRedshiftProxy(plugin.MaxCreator):
+    identifier = "io.openpype.creators.max.redshiftproxy"
+    label = "Redshift Proxy"
+    family = "redshiftproxy"
+    icon = "gear"
+
+    def create(self, subset_name, instance_data, pre_create_data):
+        from pymxs import runtime as rt
+        sel_obj = list(rt.selection)
```

Comment:
Shouldn't the selection only be used whenever `Use selection` pre create checkbox was set?

Reply:
i can set `sel_obj = list(rt.selection) or list(self.selected_node)`, hopefully it won't error out

Reply:
Actually, you don't need to do anything with the selection here, right?
It's already done in the [parent class](https://github.com/ynput/OpenPype/blob/9d3b788fb40137e651273f104682b31f907f2550/openpype/hosts/max/api/plugin.py#L52-L64).

Reply:
If my previous comment is correct you don't need to implement the `create` method on this class here at all since the parent class already creates the correct implementation for you.